### ISSUE TITLE
size of SA-V dataset

### ISSUE THREAD

**Issue by shutianXD** [Vujas-Eteph/DatasetHarvester#5]
Hi, I'd like to inquire you about the approximate size of SA-V dataset in GB. You mentioned that downloading SA-V dataset took you about 3h30min, so it must be a large dataset, right?

**Comment by Vujas-Eteph** [comment]
Hi @shutianXD, 

the total size for the dataset is 453 GB and is distributed as follow: about 432 GB for the training set, 14 GB for the validation set and 16 GB for the test set.

Hope this answers your concerns ;)

There is a way to speed up the process using multiprocessing (ref. to issue #1), but I haven't got the time to implement it.
